storage/innobase/include/btr0btr.ic

```diff
@@ -148,7 +148,7 @@ Gets the node level field in an index page.
 @return level, leaf level == 0 */
 UNIV_INLINE
 ulint
-btr_page_get_level_low(
+btr_page_get_level(
 /*===================*/
 	const page_t*	page)	/*!< in: index page */
```

Comment:
Remove the `/*====*/` uglification, move the paramter to the same line (with space, no TAB) and add a `@param` comment.
Actually, move the whole function to the .h file. I would like to get rid of the .ic files; they are always only included from the single .h file.